docs/internals.md

```diff
@@ -0,0 +1,38 @@
+# Internals
```

Comment:
Not needed since there's nothing in this repository that is described in this file.

Reply:
removed